source/_posts/2019-01-18-release-v11.7.0.md

```diff
@@ -0,0 +1,374 @@
+---
+category: articles
+title: Node v11.7.0 (현재 버전)
+author: Myles Borins
+ref: Node v11.7.0 (Current)
+refurl: https://nodejs.org/en/blog/release/v11.7.0
+translator: freenice12
+---
+
+<!--
+### Notable Changes
+
+* **compression / zlib**:
+  * Added brotli support (Anna Henningsen and Zach Vacura) [#24938](https://github.com/nodejs/node/pull/24938)
+* **console**:
+  * Added `inspectOptions` option (Ruben Bridgewater) [#24978](https://github.com/nodejs/node/pull/24978)
+* **crypto**:
+  * Always accept private keys as public keys (Tobias Nießen) [#25217](https://github.com/nodejs/node/pull/25217)
+* **deps**:
+  * Upgrade npm to v6.5.0 (Jordan Harband) [#25234](https://github.com/nodejs/node/pull/25234)
+* **fs**:
+  * Use internalBinding('fs') internally instead of process.binding('fs') (Masashi Hirano) [#22478](https://github.com/nodejs/node/pull/22478)
+* **http(s)**:
+  * Support overriding http\\s.globalAgent (Roy Sommer) [#25170](https://github.com/nodejs/node/pull/25170)
+* **util**:
+  * Inspect ArrayBuffers contents closely (Ruben Bridgewater) [#25006](https://github.com/nodejs/node/pull/25006)
+* **worker**:
+  * Expose workers by default and remove `--experimental-worker` flag (Anna Henningsen) [#25361](https://github.com/nodejs/node/pull/25361)
+-->
+
+### 주요 변경사항
+
+* **compression / zlib**:
+  * brotli 지원이 추가되었습니다. (Anna Henningsen and Zach Vacura) [#24938](https://github.com/nodejs/node/pull/24938)
+* **console**:
+  * `inspectOptions` 옵션이 추가되었습니다. (Ruben Bridgewater) [#24978](https://github.com/nodejs/node/pull/24978)
+* **crypto**:
+  * 항상 공개키로 비공개 키를 수락합니다. (Tobias Nießen) [#25217](https://github.com/nodejs/node/pull/25217)
```

Comment:
[PR의 내용](https://github.com/nodejs/node/pull/25217/files#diff-29db77451cd94ef9fbab42fd5b7499d5R1415)을 보면 `공개키 대신 비공개키를 받는다`로 보입니다. 

> Because public keys can be derived from private keys, a private key may be passed instead of a public key.

> Some APIs already accept private keys instead of public keys. This changes all relevant crypto APIs to do so.